pygmt/tests/test_blockm.py

```diff
@@ -68,7 +68,6 @@ def test_blockmean_wrong_kind_of_input_table_grid(dataframe):
     Run blockmean using table input that is not a pandas.DataFrame or file but a grid.
     """
     invalid_table = dataframe.bathymetry.to_xarray()
-    assert data_kind(invalid_table) == "grid"
```

Comment:
These `assert`s are already covered by the `data_kind` doctests and can be removed.

Reply:
I'd prefer to keep the asserts intact because `ci_doctests.yml` only runs weekly, not on every Pull Request, so it will be harder to check if refactors to `data_kind` broke something.

Reply:
These doctests are always executed in the "Tests" workflow. See https://github.com/GenericMappingTools/pygmt/actions/runs/11145126496/job/30975174325?pr=3480

```
[gw0] [ 14%] PASSED ../pygmt/helpers/utils.py::pygmt.helpers.utils.data_kind 
```

Only those added to `__doctest_skip__` are skipped in the "Tests" workflow. See #2443 and https://github.com/GenericMappingTools/pygmt/issues/2286#issuecomment-1475300763 for how it works.

Reply:
Ah yes, thanks for refreshing my memory. Then ok to remove the assertions.